gengogettext.py

```diff
@@ -140,7 +148,7 @@ def quote_jobs(jobs):
     credits = 0
     for job in r['response']['jobs']:
         currency = job['currency']
-        credits += job['credits']
+        credits += float(job['credits'])
```

Comment:
Why float?

Reply:
API returns string for some reason. 
I am coercing it to `float` to be able to count the price correctly.

Reply:
Ah, finally got it. You never should use floats for money calculations. Or you will owe me a $0.333333(3)

Please use https://docs.python.org/2/library/decimal.html

Reply:
That is not critical here, but I totally agree :)